src/webapp/views/member/verify-email.mustache

```diff
@@ -1,15 +1,21 @@
-E-mail verification for Consumer Reports CCPA
+Confirm your email & phone number with Consumer Reports
 Hello,
 
-You are receiving this message because it was supplied in an application
-sent to Consumer Reports. If you are not the person who initiated that
-application, then you may ignore this message.
+Thank you for your interest in the authorized agent pilot. Once you've confirmed your email and phone number, you can sign to make Consumer Reports your authorized agent.
 
-If you are the person who initiated that application, the please click
-the following link:
+To confirm your email, click {{{url}}}
+
+
+To confirm your phone number, you will receive a code via SMS that you can enter in your web browser.
+
+Once your email and phone are confirmed, you will move onto the final step: signing a Power of Attorney on DocuSign that gives Consumer Reports permission to act as your authorized agent.
+
+Thank you for doing your part to ensure stronger data rights in California! We're thrilled to have you as part of our pilot.
+
+Best,
+Ginny and the CR Digital Lab team
+datarightsstudy@cr.org
+<a href="https://www.consumerreports.org/cro/sms/sms-text-notification-privacy-policy.htm">Privacy Policy</a>
```

Comment:
If we send this email as HTML will all the line breaks disappear? I'll have to check that - but we can definitely merge.